content/news/027/index.md

```diff
@@ -68,6 +68,28 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Hydrofoil Generation Sailing]
+
+![hgs_screen](hydrofoil1_1024.png)
+
+[Hydrofoil Generation Sailing] is a realistic sailing/foiling inshore simulator
+in development for PC/Steam that will put you in the driving seat of modern
+competitive sailing.
+
+The game is the brain child of industry veteran Stefano Casillo
+(of Assetto Corsa fame) and features a custom made 3D engine based on DirectX11
+via winapi-rs.
+
+An engine conversion to WGPU is currently under evaluation in order to
+guarantee an easier port to platform such as Steam Deck or even mobile in the
+future.
+
+Early Access release on Steam is expected in mid 2022.
+You can follow the project on [Facebook](https://www.facebook.com/HydrofoilGenerationSailing)
```

Comment:
maybe move these fb/discord to a braced list in the first sentence?

Reply:
Done.